index.js

```diff
@@ -240,9 +252,17 @@ NYC.prototype._transformFactory = function (cacheDir) {
 
 NYC.prototype._wrapRequire = function () {
   var _this = this
-  appendTransform(function (code, filename) {
+
+  function handleJs (code, filename) {
     var relFile = path.relative(_this.cwd, filename)
     return _this._maybeInstrumentSource(code, filename, relFile) || code
+  }
+
+  appendTransform(handleJs)
+
+  this.extensions.forEach(function (ext) {
+    require.extensions[ext] = require.extensions['.js']
```

Comment:
Maybe we should use [`default-require-extensions`](https://www.npmjs.com/package/default-require-extensions) here? Just in case `require.extensions['.js']` has been messed with already by some other module?